framework/Models/Code.py

```diff
@@ -340,6 +364,18 @@ def createNewInput(self,currentInput,samplerType,**kwargs):
 
     return (newInput,kwargs)
 
+  def _expandCommand(self, origCommand):
+    """
+      Function to expand a command from string to list
```

Comment:
more explanation....it also converts the windows command

Reply:
added